draft-ietf-privacypass-auth-scheme.md

```diff
@@ -140,23 +140,31 @@ in more detail.
 ## Token Challenge {#challenge}
 
 Origins send a token challenge to clients in an "WWW-Authenticate" header field
```

Comment:
I wonder if we should mention "(or "Proxy-Authenticate")

Reply:
That seems like an increase in scope, so if we do we'd probably want to run it past the list.

Reply:
Yeah we don't necessarily need to mention that — it was just alluded to in one of martin's comments